app/src/common/css/variables/colors.scss

```diff
@@ -43,3 +43,16 @@ $COLOR--organish: #fb7848;
 $COLOR--dirty-green: #5f722b;
 $COLOR--greenish-grey: #b8d0b0;
 $COLOR--go-to-stack-trace: #fafa89;
+
+$COLOR--REGEXP-symbol: #3c00c7;
```

Comment:
@DiscoElevator think that it is better to define these variables within component styles, because this is not our general RP style guide and will not be used anywhere else.